### PR TITLE
Enable historyApiFallback and browserHistory in dev mode

### PR DESCRIPTION
**Pull request by vsaarinen**
Using hashHistory with Auth0's callbacks is likely to be a pain. Doing this simplifies authentication.

TODO:
- [x] Fix test deployments

### CODE REVIEW

**Comment by vsaarinen** [comment]
Test deployments will be broken until we get rid of HTTP auth

File: webpack.config.dev.js [hunk 1]
```diff
@@ -14,6 +14,8 @@ config.devtool = 'source-map';
 // For dev server
 config.devServer = {
   publicPath: '/',
+  // For historyRouter
```
Comment (dennari) [inline webpack.config.dev.js]: should this say 'browserHistory' instead of 'historyRouter'?

Comment (vsaarinen) [inline webpack.config.dev.js]: Yes

**Comment by dennari** [comment]
:+1:

**Pull request merged**